scrapper.py

```diff
@@ -25,39 +39,136 @@ class Crawler:
     """
     Crawler implementation
     """
+
     def __init__(self, seed_urls, max_articles: int):
-        pass
+        self.seed_urls = seed_urls
+        self.max_articles = max_articles
+        self.urls = []
 
     def _extract_url(self, article_bs):
-        pass
+        """
+        Extracts urls of articles
+        """
+        articles = article_bs.find_all('div', class_='block-container')
+
+        for article in articles:
+            if len(self.urls) < self.max_articles:
+                link = article.find('a')
+                href = link['href']
+                self.urls.append(href)
```

Comment:
Before adding a new link to self.urls, it is recommended to check that you have not yet added such a URL before as many resources duplicate links multiple times. In this lab, we aim at collecting 100 unique URLs.